apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -42,12 +38,70 @@ TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a t
 ## Without branding
 
 ```jsx live
-<VideoPlayer branding={false} title="GitHub media player">
+<VideoPlayer showBranding={false} title="GitHub media player">
   <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" />
   <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
 </VideoPlayer>
 ```
 
+## With visually hidden title
+
+```jsx live
+<VideoPlayer visuallyHiddenTitle title="GitHub media player">
+  <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## With some hidden controls
+
+```jsx live
+<VideoPlayer
+  title="GitHub media player"
+  showPlayPauseButton={false}
+  showSeekControl={true}
+  showCCButton={false}
+  showMuteButton={false}
+  showVolumeControl={false}
+  showFullScreenButton={false}
+>
+  <VideoPlayer.Source
+    src="https://primer.github.io/brand/assets/example.mp4"
+    type="video/mp4"
+  />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## Hide controls when paused
+
+```jsx live
+<VideoPlayer title="GitHub media player" showControlsWhenPaused={false}>
+  <VideoPlayer.Source
+    src="https://primer.github.io/brand/assets/example.mp4"
+    type="video/mp4"
+  />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
+## Minimal
+
+```jsx live
+<VideoPlayer
+  title="GitHub media player"
+  visuallyHiddenTitle
+  showBranding={false}
+  showControlsWhenPaused={false}
+>
+  <VideoPlayer.Source
+    src="https://primer.github.io/brand/assets/example.mp4"
+    type="video/mp4"
+  />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
+
```

Comment:
Just thought of another example to add (as a follow up or before this merges): Programmatic play/pause. This is a really common use-case, and we use this regularly with Youtube's embed API, so would be great to share how to do it with this (hook). If we don't do it now, can we add to follow up tasklist please? 🙏

Reply:
I'll add it to the list, thanks!